ambry-server/src/integration-test/java/com.github.ambry.server/Verifier.java

```diff
@@ -112,6 +112,18 @@ public void run() {
                             + propertyOutput.getBlobSize());
                     throw new IllegalStateException();
                   }
+                  if (propertyOutput.getAccountId() != payload.blobProperties.getAccountId()) {
+                    System.out.println(
```

Comment:
Server integration tests are also doing SOPs and not logger. Will fix it in a different PR.